src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1675,12 +1674,32 @@ void Graph::EnforceInferencePrecision() {
                 if (parent->getType() == Type::Input && one_of(node->getType(), Type::Eltwise, Type::Subgraph))
                     return true;
 
+                if (node->getType() == Type::Convert) {
+                    auto dstPrecision = node->getOriginalOutputPrecisionAtPort(0);
+                    if (dstPrecision.is_integral_number() && dstPrecision.bitwidth() > 8) {
```

Comment:
I would agree that condition generalization looks very logical. however it doesn't solve the issue in general. Proper precision mark-up required graph traversal to find all precision sensetive node chains. 
In fact we have plans to migrate EnforcePrecision logic on ConvertPrecision pass rails which already has some logic for precision sensetive subgraph.

As for now I would recommend to simplify the condition only to cover corresponding pattern (as we already have on line 1714). It will allow to avoid undesired performance impacts.
Please also update `src/plugins/intel_cpu/tests/functional/subgraph_tests/src/convert_range.cpp` test to cover that type of pattern. We shouldn't break correct behavior during migration on ConvertPrecision.

Reply:
OK, simplified logic, test case added!